src/hostname/hostnamed.c

```diff
@@ -473,52 +479,6 @@ static int context_write_data_static_hostname(Context *c) {
         return 0;
 }
 
-static int context_write_data_machine_info(Context *c) {
```

Comment:
Why was this removed? It breaks the build as it's used further down

Reply:
Ouch. I have no idea why I did. Maybe mistake when rebase...